beetsplug/ftintitle.py

```diff
@@ -208,7 +301,14 @@ def update_metadata(
         if not drop_feat and not contains_feat(item.title, custom_words):
             feat_format = self.config["format"].as_str()
             new_format = feat_format.format(feat_part)
-            new_title = f"{item.title} {new_format}"
+            # Insert before the first bracket containing remix/edit keywords
+            bracket_pos = find_bracket_position(item.title, bracket_keywords)
```

Comment:
I'd suggest defining `find_bracket_position` as a plugin method to make it responsible for resolution of `bracket_keywords`

```suggestion
            bracket_pos = self.find_bracket_position(item.title)
```

This way, you don't need to send `bracket_keywords` in intermediate method calls.